ee/vellum_ee/workflows/display/exceptions.py

```diff
@@ -1,7 +0,0 @@
-class NodeValidationError(Exception):
```

Comment:
We need to keep this file for backwards compatibility reasons. simply re-export `NodeValidationError` in this file, adding a comment to delete this in 2.0.0